From a22e35c3c0a9b9f97a449a202960505b2a5585f6 Mon Sep 17 00:00:00 2001
From: Yurii Rashkovskii <me@yrashk.com>
Date: Sat, 17 Feb 2018 23:38:14 +0700
Subject: [PATCH] Problem: rendered-text element is already defined

Getting this error in web console:

```
Uncaught SyntaxError: Identifier 'RenderedText' has already been
declared
```

Solution: rename sit-doc's rendered-text to rendered-doc
---
 .../webapp/src/sit-doc/{rendered-text.html => rendered-doc.html}  | 8 ++++----
 sit-web/webapp/src/sit-doc/sit-doc.html                           | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)
 rename sit-web/webapp/src/sit-doc/{rendered-text.html => rendered-doc.html} (87%)

diff --git a/sit-web/webapp/src/sit-doc/rendered-text.html b/sit-web/webapp/src/sit-doc/rendered-doc.html
similarity index 87%
rename from sit-web/webapp/src/sit-doc/rendered-text.html
rename to sit-web/webapp/src/sit-doc/rendered-doc.html
index 003da3d..9ca34b0 100644
--- a/sit-web/webapp/src/sit-doc/rendered-text.html
+++ b/sit-web/webapp/src/sit-doc/rendered-doc.html
@@ -3,7 +3,7 @@
 <link rel="import" href="/bower_components/prism-element/prism-highlighter.html">
 <link rel="import" href="/bower_components/iron-ajax/iron-ajax.html">
 
-<dom-module id="rendered-text">
+<dom-module id="rendered-doc">
   <template>
     <link rel="stylesheet" href="/bower_components/prism/themes/prism.css">
 
@@ -22,8 +22,8 @@
   </template>
 
   <script>
-    class RenderedText extends Polymer.Element {
-      static get is() { return 'rendered-text'; }
+    class RenderedDoc extends Polymer.Element {
+      static get is() { return 'rendered-doc'; }
       static get properties() {
         return {
           text: {
@@ -52,6 +52,6 @@
         super.ready();
       }
     }
-    customElements.define(RenderedText.is, RenderedText);
+    customElements.define(RenderedDoc.is, RenderedDoc);
   </script>
 </dom-module>
\ No newline at end of file
diff --git a/sit-web/webapp/src/sit-doc/sit-doc.html b/sit-web/webapp/src/sit-doc/sit-doc.html
index e37fa3f..f35982c 100644
--- a/sit-web/webapp/src/sit-doc/sit-doc.html
+++ b/sit-web/webapp/src/sit-doc/sit-doc.html
@@ -4,7 +4,7 @@
 <link rel="import" href="/bower_components/paper-icon-button/paper-icon-button.html">
 <link rel="import" href="/bower_components/iron-icons/iron-icons.html">
 
-<link rel="import" href="rendered-text.html">
+<link rel="import" href="rendered-doc.html">
 
 <dom-module id="sit-doc">
   <template>
-- 
2.15.0

